kmath-tensors/src/commonMain/kotlin/space/kscience/kmath/tensors/core/DoubleTensorAlgebra.kt

```diff
@@ -833,8 +833,33 @@ public open class DoubleTensorAlgebra :
         return qTensor to rTensor
     }
 
-    override fun StructureND<Double>.svd(): Triple<DoubleTensor, DoubleTensor, DoubleTensor> =
-        svd(epsilon = 1e-10)
+    override fun StructureND<Double>.svd(): Triple<DoubleTensor, DoubleTensor, DoubleTensor> {
```

Comment:
you should define a `svdGolubKahan` function and simply call it from the interface `svd`

Reply:
And `svdGolubKahan` should have all the parameters with default values for the interface call with `svd`

Reply:
1) ok 2) added parameters iterations and epsilon